krm-functions/scripts/build-local.sh

```diff
@@ -0,0 +1,30 @@
+#! /bin/bash
+#
+#  Copyright 2022 The Kubernetes Authors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+# TODO: When the release pipeline and docker image registry are set up,
+#  put the correct registry prefix in front of the function image name here.
+#  Currently, this will build an image named `docker.io/sig-cli/${FUNCTION NAME}:unstable`.
```

Comment:
What is the container registry that SIG-CLI owns?

Reply:
I'll do some investigation. IIUC there is not one container registry that SIG-CLI owns but there is some infra we can set up to create a community-owned registry. Kustomize has a way of publishing docker images so I will take a look at that. It may relate to some work with setting up the release pipeline, I'll get back to this thread when I know more.

Reply:
For kustomize, the image names look like `gcr.io/k8s-staging-kustomize/kustomize` and are hosted here: https://console.cloud.google.com/gcr/images/k8s-staging-kustomize/global/kustomize?project=k8s-staging-kustomize

We can do something similar for these functions. I will look into creating a GCP project for us.

Reply:
Yes, please investigate how to set up container registry owned by cncf.

Reply:
Should we update it here given we have our own registry?

Reply:
done, local images will be named `us.gcr.io/k8s-artifacts-prod/krm-functions/{function name}:unstable`